client_code/navigation.py

```diff
@@ -46,6 +46,12 @@ def open_form(form_name, full_width=False):
     get_open_form().content_panel.clear()
     get_open_form().content_panel.add_component(form, full_width_row=full_width)
 
+    
+def emulate_link_click(target):
+    """Emulate clicking a menu link"""
+    matching_links = [link for link in _links if link.tag.target == target]
+    matching_links[0].raise_event('click')
+    
```

Comment:
A nicer error when there are no matching links. 

You can also do a loop and break out when you find the first one. And use the no break syntax. 

```python
for link in links:
    if condition:
        break
else: # no break
    raise ...
link.raise_event('click')
```

Re name of the function. `go_to`or `navigate_to`?

Reply:
I implemented this, changed name to go_to, and also tried adding a more helpful error message to the get_form function.